NuRadioReco/examples/cr_efficiency_analysis/2_threshold_final.py

```diff
@@ -0,0 +1,270 @@
+import numpy as np
```

Comment:
ya, again the problem that you want to have a script which is as fast as possible. Maybe I should remove the fast option, because Im so only one to calculate that much iteration with an envelope trigger